src/components/Details/Details.tsx

```diff
@@ -52,7 +52,10 @@ export const Details: React.FC<SelectedMarketProps> = ({
   });
 
   return (
-    <div className='container'>
+    <>
+    {(!error && !openSeason) && <h2>Loading...</h2>}
+    {error && <h2>{error}</h2>} 
+    {!error &&  openSeason && <div className='container'>
```

Comment:
looks like a dbl space in `&&  openSeason`

Reply:
I have prettier, so during refactoring today all of the syntax/spacing/quote issues will go away